app/service/form_task_list_service.rb

```diff
@@ -40,7 +40,10 @@ def section_2_tasks
   end
 
   def section_3_tasks
-    [{ task_name: I18n.t("forms.task_lists.section_3.privacy_policy"), path: privacy_policy_path(@form.id) }]
+    [
+      { task_name: I18n.t("forms.task_lists.section_3.privacy_policy"), path: privacy_policy_path(@form.id) },
+      { task_name: I18n.t("forms.task_lists.section_3.contact_details"), path: contact_details_path(@form.id) },
```

Comment:
Could we add a test for this extra task?

Reply:
Added